src/fuzzy.js

```diff
@@ -8,6 +10,52 @@ export const sortKind = {
 	bestMatch: "bestMatch",
 };
 
+// Normalize Hangul consonant codes separated from phonemes.
+function normalizeKorCharCode(charCode) {
+	// The consonant code of Hangul, which is separated from the phoneme, is different from the consonant code.
+	const koCharCodes = {
```

Comment:
In the case of Hangul, even if it is the same character, the Unicode value is different when only consonants are input and when consonants are extracted by separating the consonants of the completed character.

For example
'ㄱ' and 'ᄀ' have the same shape but different Unicode values.(12593, 4352)
```ts
console.log("ㄱ".normalize("NFD").split("")[0].charCodeAt(0)); // "ㄱ" -> 12593
console.log("가".normalize("NFD").split("")[0].charCodeAt(0)); // "가(ㄱ,ㅏ)" -> 4352
```